script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_2_2.txt

```diff
@@ -44,12 +44,12 @@ But still――― Even a person like me can say what I don't want to do.
 "―――Shiki?"
 "...Arcueid. I don't understand what you're talking about.
 No matter how much of a monster he is, no matter how powerless I am, my responsibility is still the same. As I thought, I let many people die, and I don't want to see anyone else meet that fate.
-...That's why I can't run away by myself after all this time."
+...That's why I can't run away by myself after all this."
 <Page12>
 I look up at the ceiling and take a deep breath.
-Fortunately, my resolve became stronger as a result.
+Fortunately, my resolve became stronger as a result.//to be rechecked
```

Comment:
Nothing wrong translation wise.

I'd cut out fortunately entirely and say 'My resolve strengthens as a result'